pkg/router/router.go

```diff
@@ -37,15 +37,29 @@ func Setup() *gin.Engine {
 		auth.POST("/refresh", authMW.MiddlewareFunc(), authMW.RefreshHandler)
 	}
 
+	asa := r.Group("/asa", authMW.MiddlewareFunc())
```

Comment:
I didn't understand this asa router group. Do we need this?

Reply:
I forgotten remove the block when I test, I use. I fixed [e8d02d9](https://github.com/pass-wall/passwall-server/pull/54/commits/e8d02d94beabd45576a576440e36df8fda6ec6cd)